### DISCUSSION TITLE
自定义元素字体设置问题

### DISCUSSION THREAD

**Discussion by huangzhang-ops** [wangeditor-next/wangEditor-next#427] [General]
## Bug Description

https://github.com/wangeditor-team/wangEditor-plugin-upload-attachment/blob/main/example/index.ts
这个demo运行之后，添加里面的元素。
还有我想修改字体颜色和背景颜色如何修改呢？

例如：（我制定的元素+原生的文本）这些选中之后。在菜单栏上修改背景颜色等属性如何同步到我制定的元素

## What did you expect to happen?

*Please enter the details...*

## System, Browser, and Version

- Operating System
- Browser and Version

## wangEditor-next Version

*Please enter the details...*

## Can the demo reproduce the bug?

Yes/No

- Demo: https://cycleccc.github.io/demo/

## Online Demo

 - SandBox url: https://stackblitz.com/@cycleccc/collections/wangeditor

## Steps to Reproduce at Minimum Cost

- Step 1
- Step 2
- Step 3

**Comment by cycleccc** [comment]
参考 table 表格，表格内部的文字就是自定义元素中的文字。

**Comment by cycleccc** [comment]
该问题与已完成的 #417 相同，结论是参考 table 的自定义元素内部文本实现。本讨论现关闭。
